code/game/objects/effects/alien_acid.dm

```diff
@@ -73,9 +73,9 @@
 /obj/effect/acid/alien/process()
 	. = ..()
 	if(.)
-		if(prob(45))
+		if(DT_PROB(25, SSOBJ_DT))
 			playsound(loc, 'sound/items/welder.ogg', 100, TRUE)
-		target_strength--
+		target_strength -= SSOBJ_DT * 0.5
```

Comment:
Probably round given the switch below. (or changing that)

Reply:
Done